core-common/src/main/java/org/zalando/nakadi/domain/ConsumerTag.java

```diff
@@ -0,0 +1,21 @@
+package org.zalando.nakadi.domain;
+
+
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+public enum ConsumerTag {
+    SUBSCRIPTION_ID;
```

Comment:
**Meta:** I think it would be more general to support other (potential future) use cases in this enum (and the `KafkaConsumerTagSerializer` helper class) not limited to "consumer tag" semantics but to general "message tag/header".

With that in mind, maybe rename the enum (and the helper class) to smt. like `(Message)HeaderTag`? The `SUBSCRIPTION_ID` would become more specific then, like `CONSUMER_SUBSCRIPTION_ID`.